common/src/main/java/org/opensearch/ml/common/transport/model/MLUpdateModelInput.java

```diff
@@ -182,17 +207,23 @@ public static MLUpdateModelInput parse(XContentParser parser) throws IOException
                 case MODEL_GROUP_ID_FIELD:
                     modelGroupId = parser.text();
                     break;
+                case IS_ENABLED_FIELD:
+                    isEnabled = parser.booleanValue();
+                    break;
+                case MODEL_RATE_LIMITER_CONFIG_FIELD:
+                    modelRateLimiterConfig = MLRateLimiter.parse(parser);
+                    break;
                 case MODEL_CONFIG_FIELD:
                     modelConfig = TextEmbeddingModelConfig.parse(parser);
                     break;
-                case CONNECTOR_FIELD:
-                    connector = Connector.createConnector(parser);
+                case UPDATED_CONNECTOR_FIELD:
```

Comment:
User can never input updated connector, this is constructed by our code internally. Please remove this to avoid confusion.

Reply:
Ack